src/bundle/Resources/public/scss/_alerts.scss

```diff
@@ -1,12 +1,35 @@
 .ibexa-alert {
+    $self: &;
+
+    min-height: calculateRem(48px);
     display: grid;
     grid-template: 'icon content actions close-btn';
-    grid-template-columns: calculateRem(32px) 1fr auto calculateRem(32px);
-    grid-column-gap: calculateRem(8px);
-    padding: calculateRem(16px) calculateRem(19px);
+    grid-template-columns: auto 1fr auto auto;
+    padding: calculateRem(8px) calculateRem(8px);
     border-radius: 0 $ibexa-border-radius $ibexa-border-radius 0;
-    border-width: 0 0 0 calculateRem(3px);
+    border-width: 0 0 0 calculateRem(4px);
     border-left-style: solid;
+    align-items: center;
+
+    &--small {
+        min-height: calculateRem(32px);
+        padding: 0 calculateRem(8px);
+
+        #{$self} {
+            &__title {
+                font-size: calculateRem(12px);
+            }
+
+            &__icon-wrapper {
+                margin-top: calculateRem(9px); // Fixing position for rows
+
+                .ibexa-icon {
+                    width: calculateRem(12px);
```

Comment:
That's `.ibexa-icon--tiny-small` modifier, why not use it instead of extra styling here?